tensorflow/tools/ci_build/builds/configured

```diff
@@ -28,6 +28,10 @@ set -e
 CONTAINER_TYPE=$( echo "$1" | tr '[:upper:]' '[:lower:]' )
 shift 1
 
+# Enable support for MKL
+export TF_NEED_MKL=1
```

Comment:
This is the problem. If you do it here, the `configure` script on root won't ask the y/N question about MKL and just assumes it is 1.